library/proc_macro/src/lib.rs

```diff
@@ -1351,6 +1352,15 @@ impl Literal {
         Literal::new(bridge::LitKind::ByteStr, &string, None)
     }
 
+    /// C string literal.
+    #[stable(feature = "c_str_literals", since = "1.76.0")]
```

Comment:
Ok, I'm not sure how to do that for this particular change. There's already a tracking issue #105723 which was closed without this API but this API is listed on that tracking issue. I'm only finding info in the rustc dev guide about creating a tracking issue by making an RFC PR, but is that required here since there's already an accepted PR and closed tracking issue? Apologies for all the questions, this is my first contribution :)

Reply:
You can just create a new issue on this repo, there is a template for "Library tracking issue" that you can use. Then just change the `#[stable]` in this PR to `#[unstable(feature = "proc_macro_c_str", issue = "XXXX")]`.

Reply:
Ok, created one #119750. I think #105723 is about to get reverted by #119528, I don't think that changes this PR since it'll need to get stabilized anyway.